games-action/tuxracer/patches/tuxracer-0.61.patch

```diff
@@ -0,0 +1,20 @@
+--- tuxracer-0.61/src/game_config.c.org	2016-04-25 18:59:37.147587072 +0200
++++ tuxracer-0.61/src/game_config.c	2016-04-25 18:59:23.122945536 +0200
+@@ -79,7 +79,7 @@
+ #  if defined( WIN32 )
+ #    define DATA_DIR "."
+ #  else
+-#    define DATA_DIR "/usr/local/share/tuxracer"
++#    define DATA_DIR "/boot/home/config/apps/TuxRacer/data"
```

Comment:
Something like this should work: $portPackageLinksDir/tuxracer_data/apps/TuxRacer/data
$portPackageLinksDir is to be replaced with a macro at build time.